covfee/client/tasks/continuous_annotation/index.tsx

```diff
@@ -645,6 +650,33 @@ const ContinuousAnnotationTask: React.FC<Props> = (props) => {
 
   return (
     <form>
+      {showInitialTaskIntructions && (
+        <Modal
+          title={"Task instructions"}
```

Comment:
The task already has an "instructions" panel. So would be good to use another word here: perhaps "Reminder", or "Overview" or "variant" ... not sure what is more appropriate.

Reply:
Fixed in 0772750d37dc1dcc5c873945250f37d1afe68936